README.md

```diff
@@ -191,6 +195,30 @@ query {
 -   Incoming queries are validated against the GraphQL schema. If the query is invalid, a response with status code `400` is returned along with an array of GraphQL Errors that were found.
 -   To avoid disrupting server activity, errors thrown during the analysis and rate-limiting of the query are logged and the request is passed onto the next piece of middleware in the chain.
 
+## <a name="comlpexity"></a> Complexity Analysis
+
+This package exposes 3 additional functionalities
+
+1. #### `typeWeightsFromSchema` | create the type weight object from the schema for complexity analysis
+   - `schema: GraphQLSchema` | GQL schema object
+   - `typeWeightsConfig: TypeWeightConfig = defaultTypeWeightsConfig`
+   - `enforceBoundedLists = false`
+   - returns `TypeWeightObject`  
+
+2. #### `ComplexityAnalysis` | calculate the complexity of the query based on the type weights and variables
+   - `typeWeights: TypeWeightObject`
+   - `variables: Variables` | varibales on request
+   - returns Class
+      - `processQuery(queryAST: DocumentNode): number`
```

Comment:
if returning class, we may want it to be `ProcessQuery`

Reply:
This is a method on the class. I didn't know how to specify that. Obviously its not clear enough.